disperser/apiserver/server.go

```diff
@@ -778,6 +795,15 @@ func (s *DispersalServer) Start(ctx context.Context) error {
 	return nil
 }
 
+func (s *DispersalServer) LoadAllowlist() {
+	al, err := ReadAllowlistFromFile(s.rateConfig.AllowlistFile)
+	if err != nil {
+		s.logger.Error("failed to load allowlist", "err", err)
```

Comment:
So if it fails to load allowlist, it will just keep the old values?

Reply:
Yes!